snowplow/iglu-client-embedded/schemas/com.metabase/embed_share/jsonschema/1-0-0

```diff
@@ -23,7 +23,7 @@
       "type": [
         "boolean",
         "null"
-      ],
```

Comment:
this was actually one of the reasons tests were totally failing

Reply:
Yeah this is a malformed JSON, not sure why there is one here at first.